llm/gpt-2/gpt2-train.yaml

```diff
@@ -21,11 +22,11 @@ resources:
 file_mounts:
   ~/.cache/huggingface:
     name: $BUCKET_NAME
+    store: $BUCKET_STORE
     mode: COPY
 
 setup: |
   cd ~
-  pip install tqdm tiktoken requests datasets
```

Comment:
is this intended?

Reply:
Yes, it is not needed for training stage, but only for the data processing stage : )